next/utils/forms.ts

```diff
@@ -441,6 +445,31 @@ interface Callbacks {
   onInit?: () => Promise<any>
 }
 
+export const useFormRJSFContext = (eform: EFormValue, formId?: string): FormRJSFContext => {
+  const { schema } = eform
+  const [bucketFolderName, setBucketFolderName] = useState<string>()
+
+  useEffect(() => {
+    if (schema.pospID && formId) {
+      setBucketFolderName(`/${String(schema.pospID)}/${formId}`)
+    }
+  }, [schema, formId, setBucketFolderName])
+
+  return {
+    bucketFolderName
+  }
+}
+
+export const useFormContext = (eform: EFormValue, formId?: string) => {
```

Comment:
```suggestion
export const useFormRJSFContextMemo = (eform: EFormValue, formId?: string) => {
```

to (at least slightly) differentiate from a use*Context hook expecting a provider in the app hierarchy